mx.py

```diff
@@ -9728,7 +9728,8 @@ def processDep(dep, edge):
             libFiles.append(path)
 
     for dep in sorted(projectDeps):
-        out.element('classpathentry', {'combineaccessrules' : 'false', 'exported' : 'true', 'kind' : 'src', 'path' : '/' + dep.name})
+        if not isinstance(dep, NativeProject):
```

Comment:
There are a bunch of type test methods defined on the `Dependency` class. Please change the above to use `isNativeProject()` instead.